src/app/shared/interfaces/movies.ts

```diff
@@ -0,0 +1,25 @@
+
+class Results {
+    "vote_count": number;
+    "id": number;
+    "video": boolean;
+    "vote_average": number;
+    "title": string;
+    "popularity": number;
+    "poster_path": string;
+    "original_language": string;
+    "original_title": string;
+    "genre_ids": number[];
+    "backdrop_path": string;
+    "adult": boolean;
+    "overview": string;
+    "release_date": string
+}
+
+export class Movies<Object> {
```

Comment:
is it enough to be just Movies without <Object>

Reply:
Done